sources/install/package_web.sh

```diff
@@ -743,6 +745,11 @@ function install_sqlmap() {
     colorecho "Installing sqlmap"
     git -C /opt/tools/ clone --depth 1 https://github.com/sqlmapproject/sqlmap.git
     ln -s "/opt/tools/sqlmap/sqlmap.py" /opt/tools/bin/sqlmap
+    echo '#!/usr/bin/env python3' > tempfile.py
+    tail -n +2 /opt/tools/sqlmap/sqlmap.py >> tempfile.py
+    mv tempfile.py /opt/tools/sqlmap/sqlmap.py 
+    chmod +x /opt/tools/sqlmap/sqlmap.py 
+    rm -f ./tempfile.py
```

Comment:
Can you explain this change ?

Reply:
sqlmap is executed without specifying the python interpreter and as the version of python is not specified this will call python2 and therefore I modify the first line of the `sqlmap.py` from `#!/usr/bin/env python` to ` #!/usr/bin/env python3`

Reply:
Wouldn't it be better to default `python` to `python3`? If we opt for this change, the hot patches like this one will need to be removed